### DISCUSSION TITLE
2024-huangshan/

### DISCUSSION THREAD

**Discussion by giscus** [n0vad3v/blog-comments#15] [Ideas]
# 2024-huangshan/

上一次来黄山可是上一次了，接近 20 年前，回顾着老照片再次登上黄山，整体给我的感觉便是——不像最近多年来火热的「网红旅游区」，黄山似乎整体保持了一个很正常的状态（当然，也有可能因为是在山上，没法网红起来。

https://nova.moe/2024-huangshan/

**Comment by jiangKerman** [comment]
快带keshane回重庆玩，我们这儿山多酒店便宜😄

**Comment by hunterlxt** [comment]
很好，期待FK7开到伊犁的那天😍

**Comment by spencercjh** [comment]
我在 23 年换工作的间隙徒步从山脚下徒步爬的黄山。5 点出发，一路登顶光明顶和莲花峰，随后游客高峰期到来后就坐缆车下山了。开着 CC 猎装从上海出发，周游黄山->衢州->龙游->桐庐，不上班在外面真开心。
